src/test/kotlin/uk/gov/justice/digital/hmpps/approvedpremisesapi/integration/PlacementApplicationsTest.kt

```diff
@@ -900,7 +900,7 @@ class PlacementApplicationsTest : IntegrationTestBase() {
 
                       val createdPlacementApplication = createdPlacementRequests[0]
 
-                      assertThat(createdPlacementApplication.allocatedToUser.id).isIn(listOf(matcher1.id, matcher2.id))
+                      assertThat(createdPlacementApplication.allocatedToUser!!.id).isIn(listOf(matcher1.id, matcher2.id))
```

Comment:
It might be worth adding:
```
assertThat(createdPlacementApplication.allocatedToUser).isNotNull
```
as an additional assertion before this one so that if it is null the test fails cleanly instead of through a `NullPointerException`.